mailchimp3/helpers.py

```diff
@@ -16,17 +17,27 @@
 def get_subscriber_hash(member_email):
     """
     The MD5 hash of the lowercase version of the list member's email.
-    Uses as memeber_id
+    Used as subscriber_hash
 
     :param member_email: The member's email address
     :type member_email: :py:class:`str`
     :returns: The md5 hash in hex
     :rtype: :py:class:`str`
     """
+    if not re.search('@', member_email):
+        raise SyntaxError('String passed is not a valid email address')
```

Comment:
I'd use `ValueError` here. `SyntaxError` is for Python syntax and usually raised when importing modules.